src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -738,13 +732,6 @@ public void SendData(INodePacket packet)
                     writeStream.Position = 1;
                     writeStream.Write(BitConverter.GetBytes((int)writeStream.Length - 5), 0, 4);
 
-#if FALSE
-                    if (trace) // Avoid method call
-                    {
-                        CommunicationsUtilities.Trace(nodeId, "Sending Packet of type {0} with length {1}", packet.Type.ToString(), writeStream.Length - 5);
```

Comment:
Hmm, this could actually be useful in debugging. Currently, node communication tracing only logs node connection messages instead of all communication, and I often had to println packets on my own. @rainersigwald, would you like to enable logging of all packets here, or just delete it and maybe someday devise a better communication log infrastructure later?

Reply:
On the other hand I see that there's no symmetric code in other places to enable full packet tracking, so yeah, just delete this one as well.